admin/app/components/solidus_admin/main_nav_component.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+module SolidusAdmin
+  # Renders the main navigation of Solidus Admin.
+  class MainNavComponent < BaseComponent
+    def initialize(main_nav_item_component: component("main_nav_item"), items: container.within_namespace("main_nav"))
+      @main_nav_item_component = main_nav_item_component
+      @items = items
+    end
+
+    erb_template <<~ERB
+      <nav>
```

Comment:
Is tailwind able to catch the classes of these HTML elements even if they are in the .rb file?

Reply:
We should be fine if we add the `components` directory to the Tailwind configuration. [Tailwind uses ](https://tailwindcss.com/docs/content-configuration#class-detection-in-depth)a very simple regexp matching to extract the needed classes.

Reply:
FYI, I rebased from the admin branch, which now contains tailwindcss, and pushed a new commit adding the component directories to the list of scanned paths.

Reply:
💯